sdk/keyvault/Azure.Security.KeyVault.Certificates/README.md

```diff
@@ -138,6 +138,20 @@ Console.WriteLine(deletedCert.ScheduledPurgeDate);
 await client.PurgeDeletedCertificateAsync("MyCertificate");
 ```
 
+### List Certificates
+`GetCertificatesAsync` Enumerates the certificates in the vault, returning select properties of the
```

Comment:
Should be "enumerates", and we also don't explicitly mention the "Async" suffix for methods unless necessary. This is a naming convention in .NET but we refer to the method names as more of a family, i.e. "GetCertificates" for both `GetCertificates` and `GetCertificatesAsync` - they are sync/async alternatives to each other.